src/portal_visualization/builders/anndata_builders.py

```diff
@@ -71,6 +73,48 @@ def get_conf_cells(self, marker=None):
         cell_set_obs.append("leiden")
         cell_set_obs_names.append("Leiden")
         gene_alias = 'var/hugo_symbol' if 'var' in z and 'hugo_symbol' in z['var'] else None
+        if (gene_alias and marker is not None):
+            # If user has indicated a marker gene in parameters and we have a hugo_symbol mapping,
+            # then we need to convert it to the proper underlying ensembl ID for the dataset
+            # in order for the views to reflect the correct gene.
+
+            obs_attrs = dict(z["obs"].attrs)
```

Comment:
```suggestion
            obs_attrs = z["obs"].attrs.asdict()
```
Preference for the built-in method: https://zarr.readthedocs.io/en/stable/_modules/zarr/attrs.html#Attributes.asdict